index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <script src="https://cdn.tailwindcss.com"></script>
+    <script>
+      tailwind.config = {
+        theme: { extend: { colors: { clifford: "#da373d" } } },
+      };
+    </script>
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main-doc" class="flex flex-row">
+      
+      <div class="basis-1/4">
+        <nav
+          id="navbar"
+          class="flex flex-col lg:fixed h-screen lg:w-1/4 z-10 sm:max-h-23 sm:absolute sm:w-full min-w-290 top-0 lg:overflow-hidden bg-gray-400 space-x-2"
+        >
+          <header class="font-bold lg:text-xl md:text-xl sm:text-normal">
```

Comment:
I think it would be better to use semantic tags <h1>

Reply:
- Same here kindly remember to always include an explanation, a "why". Also please try to include some emojis It is always a good idea. :eyes:

Reply:
Great job adding some emojis :raised_hands: ! But also please be careful of their usability as some can be easily misinterpreted :eyes: :warning: !

Reply:
Is that okay? Do I need to add something?